vme/src/affect.cpp

```diff
@@ -212,16 +212,16 @@ void affect_clear_unit(unit_data *unit)
 
     /* Some affects may not be destroyed at first attempt if it would */
     /* cause an overflow, therefore do several attemps to destroy     */
-    for (i = 0; UNIT_AFFECTED(unit) && (i < 5); i++)
+    for (i = 0; unit->getUnitAffectedType() && (i < 5); i++)
     {
-        for (taf1 = UNIT_AFFECTED(unit); taf1; taf1 = taf2)
+        for (taf1 = unit->getUnitAffectedType(); taf1; taf1 = taf2)
         {
             taf2 = taf1->getNext();
             destroy_affect(taf1);
         }
     }
 
-    if (UNIT_AFFECTED(unit))
+    if (unit->getUnitAffectedType())
```

Comment:
If you're renaming a few things, maybe drop the "Type" at the end of this so it's just getUnitAffected()

Reply:
Done